src/pages/chainlink-nodes/configuring-nodes.mdx

```diff
@@ -0,0 +1,127 @@
+---
+layout: ../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Configuring Nodes"
+---
+
+Starting with Chainlink v2.0.0, TOML configuration is the only supported configuration method. You can switch to TOML format using the [Chainlink v1.13.0 image](https://hub.docker.com/r/smartcontract/chainlink/tags). After your Chainlink node is running stable on v1.13.0, you can continue to use the TOML config on future images where support for `.env` configs are no longer supported.
+
+You can see the available config options on the [Node Config](/chainlink-nodes/v1/node-config) and [Node Secrets](/chainlink-nodes/v1/secrets-config) pages.
+
+## Migrating from environment variables to TOML
+
+Before you begin, update your Chainlink Node to v1.13.0 and ensure that it operates as expected. This is the last version that supports environment variable configuration, and the best version to use for TOML migration before you update the node to future versions.
+
+### Export your current config
+
+You can generate the `config.toml` file using the `chainlink config dump` command. This guide shows you how to do this for a Chainlink node running in a Docker container:
+
+1. Open an interactive shell inside the Docker container that runs your node:
+
+   ```shell
+   docker exec -it chainlink bash
+   ```
+
+1. Log in to the Chainlink CLI with an account that has admin access:
+
+   ```shell
+   chainlink admin login
+   ```
+
+1. Export the current config to a `config.toml` file:
+
+   ```shell
+   chainlink config dump > config.toml
+   ```
+
+1. Log out of the Chainlink CLI and close the shell on the Docker container.
+
+   ```shell
+   chainlink admin logout && exit
+   ```
+
+1. Make a new directory on the host system to store your config files. Then, `cd` to that directory. This directory will be mounted to Docker when you run the node using the TOML config.
+
+   ```shell
+   mkdir ~/.chainlink-toml && cd ~/.chainlink-toml
+   ```
+
+1. Write the config file from the container to your host:
+
+   ```shell
+   docker exec -it chainlink cat /home/chainlink/config.toml > ./config.toml
+   ```
+
+1. Create a `secrets.toml` file with the minimum required secrets, which are `[Database]` and `[Password]`. If you are working on a test node, you can use `?sslmode=disable` in the database `URL`:
```

Comment:
Should we also add a note about adding `[Explorer]` and any other relevant sections from the [SECRETS.md](https://github.com/smartcontractkit/chainlink/blob/v1.13.0/docs/SECRETS.md#explorer) doc? I see the doc is mentioned below, but I wonder if it'd be good to explicitly call this one out.

Reply:
Added. Does the description make sense?

Reply:
Perfect, thanks!